### PR TITLE
MNT: Re-rendered with conda-smithy 2.0.1

### PR DESCRIPTION
**Pull request by kain88-de**
updates numpy to 1.12

### CODE REVIEW

**Comment by kain88-de** [comment]
seems like the  python 3.5 and numpy 1.12 options fails. But that's hard to say since the output just ends with a compiler message.

**Comment by kain88-de** [comment]
```
===== testing package: astropy-1.3-np112py36_0 =====

import: 'astropy'

Traceback (most recent call last):

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/test_tmp/run_test.py", line 27, in <module>

    import astropy

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/__init__.py", line 116, in <module>

    _check_numpy()

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/__init__.py", line 104, in _check_numpy

    from .utils import minversion

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/utils/__init__.py", line 20, in <module>

    from .compat.odict import OrderedDict

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/utils/compat/__init__.py", line 14, in <module>

    from .numpycompat import *

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/utils/compat/numpycompat.py", line 24, in <module>

    NUMPY_LT_1_12 = not minversion('numpy', '1.12dev')

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/site-packages/astropy/utils/introspection.py", line 159, in minversion

    return parse_version(have_version) >= parse_version(version)

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/distutils/version.py", line 70, in __ge__

    c = self._cmp(other)

  File "/Users/travis/miniconda3/conda-bld/astropy_1487090318682/_t_env/lib/python3.6/distutils/version.py", line 337, in _cmp

    if self.version < other.version:

TypeError: '<' not supported between instances of 'int' and 'str'

TESTS FAILED: astropy-1.3-np112py36_0
```

there is a version check failure with numpy 1.12 due to a type error

**Comment by kain88-de** [comment]
the current version doesn't work with the new numpy. The issue is fixed in master but it we have to wait for a new release to make update this repository for numpy 1.12

https://github.com/astropy/astropy/commit/027dbfad5f0064d287e95043179eec7f9981245d

**Comment by bsipocz** [comment]
@kain88-de - waiting a bit for the new release is OK, I plan to do a 1.3.1 very soon.

**Comment by kain88-de** [comment]
@bsipocz The current version with the patch should work.  If you want to wait for a 1.3.1 release before you upgrade this package I'm OK with that too.

**Comment by bsipocz** [comment]
Lets have this if CI all green, I don't have a target date for the new release, just that it's happening soon (meaning something in a few days--weeks)

**Comment by kain88-de** [comment]
I don't think I can do much about the failing windows builds. They are hitting an internal compiler error under x64 with python 3.x.

**Comment by bsipocz** [comment]
Oh, yes that rings a bell, probably this is the same issue. We need to use the VS 2015 Update 2 for our windows CI, https://github.com/appveyor/ci/issues/890#issuecomment-229427859

**Comment by bsipocz** [comment]
hmm, maybe it's not the same issue at the end as the local appveyor here is already set to use that one...

File: appveyor.yml [hunk 1]
```diff
@@ -44,16 +64,22 @@ environment:
...
+
 
 # We always use a 64-bit machine, but can build x86 distributions
 # with the TARGET_ARCH variable.
 platform:
     - x64
 
-# Build on this OS until we can figure out how to workaround a compiler bug
-# introduced in Update 3.
-os: Visual Studio 2015 Update 2
```
Comment (kain88-de) [inline appveyor.yml]: There has been a workaround in this repo that you also have in astropy. I deleted it with the update.

@conda-forge/core Is is OK if I add the workaround in the appveyor file again?

Comment (bsipocz) [inline appveyor.yml]: I should have checked the diff ;)

Comment (kain88-de) [inline appveyor.yml]: Don't worry I should have checked it to. Would have avoided my question ;)

**Comment by kain88-de** [comment]
So Travis failed due to  some timeouts of the http requests.  I assume that they would have passed if not for that. For some reason I'm not able to restart the builds though.

**Comment by ocefpaf** [comment]
> So Travis failed due to some timeouts of the http requests. I assume that they would have passed if not for that.

I restated Travis-CI for you.

> For some reason I'm not able to restart the builds though.

If you want that ability you need to add yourself as a maintainer for this recipe.

**Comment by kain88-de** [comment]
> If you want that ability you need to add yourself as a maintainer for this recipe.

I'm OK with not having it. I was just thought I had it since travis offered me the button for a restart.

Also the tests are passing :)

**Pull request merged**